Foundation/Dictionary.swift

```diff
@@ -18,7 +18,8 @@ extension Dictionary : _ObjectTypeBridgeable {
         
         var idx = 0
         
-        self.forEach { (keyItem, valueItem) in
+        self.forEach { (arg) in
```

Comment:
This change actually came from the migrator, it may be worth fixing that to not add parentheses as I was taking the fix-its as best practice. Although I'll see what the outcome of the discussion on swift-evolution is before making any changes to this PR